README.md

```diff
@@ -1 +1,173 @@
-# portfolio
+
+<a name="readme-top"></a>
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Portfolio] <a name="about-project"></a>
+
+**[Portfolio]** takes you through how to build a portfolio webpage starting with mobile screen sizes.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML5</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>CSS3</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+This repo only takes you through how to build a portfolio webpage.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+We will take you through the process. follow these steps.
+
+### Prerequisites
+
+In order to get started, you need to be familiar with basic **git commands**, **HTML5**, **CSS3** and how to **Create a repository**.
+
+### Setup
+
+Clone this repository to your desired folder.
+
+Example commands:
+
+```sh
+  cd my-folder
+  
+  git clone git@github.com:myaccount/my-project.git
+```
+
+### Create your **files** and **folder** for your images.
+
+Example command: 
+
+```sh
+mkdir images
+
+touch index.html
+
+touch style.css
+```
+
+###  **Open** your files in your default browser. 
+
```

Comment:
[OPTIONAL]  In the setup section all you have to mention are steps and commands to run in order to get this project work in a users local machine git clone repo is enough it will generate all content of this repo

Reply:
Thanks for the feedback. It will be very helpful.